### DISCUSSION TITLE
Edit the labels of a discussion

### DISCUSSION THREAD

**Discussion by esmaeilzadeh2** [community/community#137328] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello dear friends, does anyone know how we can edit the labels after we create a new discussion to add new labels to it?
_____________________
step 1:
![111](https://github.com/user-attachments/assets/96958a93-0925-41b2-929c-8247da2955b4)
_____________________
step 2:
![222](https://github.com/user-attachments/assets/ab155dbe-7436-4fb4-b0ad-72f707e00478)
_____________________
step 3:
Click on New discussion
_____________________
step 4:
![333](https://github.com/user-attachments/assets/1f061f71-933b-4302-9976-ce8fc6fd616c)
_____________________
step 5:
![444](https://github.com/user-attachments/assets/b689c685-1109-4b9e-a2d8-703cecb879a7)
_____________________
After a new discussion is created, how can I edit its labels?

**Comment by AkshayMoolya** [comment, marked as answer]
Editing Labels for Existing GitHub Discussions:

1. Open the discussion:
   - Navigate to the repository
   - Click on the "Discussions" tab
   - Select the discussion you want to edit

2. Find the labels section:
   - Look for the gear icon (⚙️) next to "Labels" on the right side

3. Edit labels:
   - Click the gear icon to open the labels menu
   - Add new labels by selecting them from the list
   - Remove existing labels by clicking the 'x' next to them

4. Save changes:
   - Changes are saved automatically

**Reply by esmaeilzadeh2** [reply]
it was resolved Thank you!
